src/operator/tensor/elemwise_binary_broadcast_op_extended.cc

```diff
@@ -55,7 +92,32 @@ NNVM_REGISTER_OP(_backward_broadcast_maximum)
                                                               mshadow_op::lt>);
 
 MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(broadcast_minimum)
-.describe(R"code(Element-wise minimum of array elements with broadcasting.
+.describe(R"code(Returns element-wise minimum of the input arrays with broadcasting.
+
+It compares input arrays and returns the new array having the element-wise minima.
```

Comment:
See the above change (for corresponding function with max) and adapt to this function.